azure-pipelines.yml

```diff
@@ -48,24 +70,11 @@ jobs:
             script: |
               pipenv sync --dev --sequential
               pipenv run python -m spacy download en_core_web_lg
-
-        - task: Bash@3
-          displayName: 'Linting: Analyzer'
-          inputs:
-            targetType: 'inline'
-            workingDirectory: 'presidio-analyzer'
-            script: |
-              pipenv run flake8 presidio_analyzer
-
-        - task: Bash@3
-          displayName: 'Unit tests: Analyzer'
-          inputs:
-            targetType: 'inline'
-            workingDirectory: 'presidio-analyzer'
-            script: |
-              # Install pytest and run tests
-              pipenv install --dev --skip-lock pytest-azurepipelines
-              pipenv run pytest
+        
+        - template: .pipelines/templates/build-python.yml
+          parameters:
+            SERVICE: 'Analyzer'
+            WORKING_FOLDER: 'presidio-analyzer'
 
         - task: Bash@3
```

Comment:
Still required? Aren't these two last steps in the template already?

Reply:
since the dependency for analyzer and anonymizer are dofferet, I left them outside of the tempalte. Thinking of an elagant way to handle this....